doc/howtos/firewall-configuration.rst

```diff
@@ -119,3 +119,11 @@ Opening holes to firewall using this method is not absolutely mandatory.
 If you are building a device without dynamic services or applications,
 you can simply add the necessary rules to the custom ruleset that is
 loaded upon device startup.
+
+Note that if you are starting your service as non-root user (as you
+probably should be) using systemd's ``User=`` option, you'll need to
+make sure that the firewall manipulation commands are run with original
+permissions. An easy way to do that is to use
+``PermissionsStartOnly=true`` option in the service file. This works if
+you don't need to run any other ``ExecStartPre=`` or similar commands
+with the reduced non-root privileges.
```

Comment:
Em quarta-feira, 20 de abril de 2016, Geoffroy Van Cutsem <
notifications@github.com> escreveu:

> In doc/howtos/firewall-configuration.rst
> <https://github.com/ostroproject/meta-ostro/pull/124#discussion_r60389378>
> :
>
> > @@ -119,3 +119,11 @@ Opening holes to firewall using this method is not absolutely mandatory.
> >  If you are building a device without dynamic services or applications,
> >  you can simply add the necessary rules to the custom ruleset that is
> >  loaded upon device startup.
> > +
> > +Note that if you are starting your service as non-root user (as you
> > +probably should be) using systemd's ``User=`` option, you'll need to
> > +make sure that the firewall manipulation commands are run with original
> > +permissions. An easy way to do that is to use
> > +``PermissionsStartOnly=true`` option in the service file. This works if
> > +you don't need to run any other ``ExecStartPre=`` or similar commands
> > +with the reduced non-root privileges.
>
> Would that require a manual edit of the file generated by the AppFW?
>
Of course not. This will be done by the generator



> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ostroproject/meta-ostro/pull/124/files/16a0329042cef2237b9560fbf4538b6e9d36c92e#r60389378>
>


-- 
Gustavo Sverzut Barbieri
--------------------------------------
Mobile: +55 (19) 99225-2202
Contact: http://www.gustavobarbieri.com.br/contact